src/test/java/de/tum/in/www1/artemis/service/export/CourseExamExportServiceTest.java

```diff
@@ -0,0 +1,80 @@
+package de.tum.in.www1.artemis.service.export;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNoException;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.test.context.support.WithMockUser;
+
+import de.tum.in.www1.artemis.AbstractSpringIntegrationIndependentTest;
+import de.tum.in.www1.artemis.course.CourseUtilService;
+import de.tum.in.www1.artemis.repository.CourseRepository;
+import de.tum.in.www1.artemis.repository.ExamRepository;
+import de.tum.in.www1.artemis.repository.ExerciseRepository;
+import de.tum.in.www1.artemis.user.UserUtilService;
+
+class CourseExamExportServiceTest extends AbstractSpringIntegrationIndependentTest {
+
+    private static final String TEST_PREFIX = "exam_export";
+
+    @Autowired
+    private CourseUtilService courseUtilService;
+
+    @Autowired
+    private ExamRepository examRepository;
+
+    @Autowired
+    private CourseExamExportService courseExamExportService;
+
+    @Autowired
+    private UserUtilService userUtilService;
+
+    @Autowired
+    private CourseRepository courseRepository;
+
+    @Autowired
+    private ExerciseRepository exerciseRepository;
+
+    @BeforeEach
+    void setup() {
+        // setup users
+        userUtilService.addUsers(TEST_PREFIX, 5, 5, 0, 1);
```

Comment:
Can we use a smaller number of users in this test? Instantiating users takes time and lengthens the tests

Reply:
I reduced them a bit, but due to the nature of the code in the test services, I have to have a minimum number. Refactoring such code to be more flexible is out of scope.